packages/cli/test/commands/upgrade.test.ts

```diff
@@ -16,6 +16,8 @@ const TEST_TSC_VERSION = '4.5.5';
 const ORIGIN_TSC_VERSION = packageJson.devDependencies.typescript;
 let WORKING_BRANCH = '';
 
+const TEST_TIMEOUT = 100000;
```

Comment:
lets remove this line and just up the global test timeout to 100,000
https://github.com/rehearsal-js/rehearsal-js/blob/master/packages/cli/vitest.config.ts#L6

Reply:
> Why do we need to check all the config files and non app code in? Can we not just add them all to the git ignore? eg. .gitkeep, *.md, .watchmanconfig, *.yml etc.

Primarily, I wanted to keep the full app functional so I can invoke `ember <build, serve,test>` etc.

I wanted to make it a `ember new <app-name>` that way it's easier to regenerate and replace if needed. I will trim it down for or provide a regenerate script and prune script. So we can recreate this fixtures.

Reply:
Removed the files, wasn't sure how critical they were. Pruned 👍